.github/workflows/ci.yml

```diff
@@ -66,6 +66,8 @@ jobs:
       - uses: julia-actions/julia-buildpkg@latest
       - name: System info
         run: julia --project=. --color=yes -e "using BinaryBuilderBase; BinaryBuilderBase.versioninfo()"
+      - name: Install HistoricalStdlibVersions
+        run: julia --project=. --color=yes -e 'using Pkg; Pkg.add(; url="https://github.com/JuliaPackaging/HistoricalStdlibVersions.jl")'
```

Comment:
This is only while the package is being registered, no need to cut short the waiting time.